src/logger.ts

```diff
@@ -0,0 +1,32 @@
+import fetch from 'cross-fetch';
+const _debounce = require('lodash/debounce');
+
+const debouncedSendLogs = _debounce(() => sendLogs(), 1000);
+
+let collectedLogs: any[] = [];
+
+function _log(level: string, message: string, args: any = {}) {
+  const l = { level, message, ...args, _time: new Date(Date.now()).toISOString() };
```

Comment:
would it be better to have a variadic message, instead of message and args, so something like
```ts
function _log(level: string, ...message: any[]) { }

export const log = {
  info: (...message: any[]) => _log('info', ...message),
  // ...
}
```
this way a user could pass an arbitrary amount of values to log,  similar to how `console.log` works, eg: 
```ts
// message => ['foo', 'bar', 123, { other: 'value'}, [1,2,3]]
log.info('foo', 'bar', 123, { other: 'value' }, [1,2,3]); 
```

Reply:
I get the idea to be close to `console.log`, but it doesn't really work as the data is not structured and much harder to search.